modules/users/manifests/joshpedro.pp

```diff
@@ -0,0 +1,8 @@
+# Creates the joshpedro user
+class users::joshpedro {
```

Comment:
Uh, looks like computer said no to your choice of username :(

users::joshpedro should have a username of the correct form
     Failure/Error: expect(user[:name]).to eq(uname_from_full), "expected username '#{user[:name]}' to be '#{uname_from_full}' based on fullname and email"
       expected username 'joshpedro' to be 'joshuapedro' based on fullname and email